python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -222,7 +224,11 @@ def submit(self, subjobconfig, master_input_sandbox):
         f = open(dirac_script_filename, 'w')
         f.write(dirac_script)
         f.close()
-        return self._common_submit(dirac_script_filename)
+        try:
+            return self._common_submit(dirac_script_filename)
+        except GangaDiracException as err:
+            logger.warning("Error during job submission: %s" % err)
+            raise GangaException(err)
```

Comment:
What version are we reviewing this was renamed to Error a while ago, does it still exist in the code in the latest form? If so I'll just let the command raise itself (bar the `finally:` method which has now been added).